cluster/ci/config/prow/plugins.yaml

```diff
@@ -943,6 +943,9 @@ config_updater:
     cluster/ci/monitoring/sinker.json:
       name: grafana-dashboard-sinker
       namespace: prow-monitoring
+    cluster/ci/monitoring/ghproxy.json:
```

Comment:
According to my experience last time, the change on `plugins.yaml` has to be merged as a separate PR before the rest.

Reply:
it should be ok. What was the problem?

Reply:
Not sure. I did not try otherwise. :)